engine/src/precompiles/blake2.rs

```diff
@@ -95,8 +95,8 @@ impl Precompile for Blake2F {
 
 #[cfg(test)]
 mod tests {
+    use super::super::utils::new_context;
```

Comment:
`super::super` seems hard to follow; probably better to go down from the root instead of up from here (same comment applies to all other places where `super::super` is used)

Reply:
It was fixed at #260